src/lang-promql/parser/parser.test.ts

```diff
@@ -114,6 +114,11 @@ describe('Scalars and scalar-to-scalar operations', () => {
       expectedValueType: ValueType.vector,
       expectedDiag: [] as Diagnostic[],
     },
+    {
+      expr: 'foo*bar',
```

Comment:
Would be worth add these tests :

`a +b`
`a+b`
`a+ b`
`a==b`

The first 3 tests are to check the space effect, the last one for binOps expression that are long than 1 char

Reply:
Yup, added.